### DISCUSSION TITLE
A newbies take on Music Assistants Areas that need improvement/enhancement

### DISCUSSION THREAD

**Discussion by wambs** [music-assistant/support#2703] [Feature requests and ideas]
First off, let me say that Music Assistant is great!! A big thank you to the developers!!

I have only used music assistant for 2 weeks. 
I know that there are discussions on specific area that need to be improved.
But I wanted to see in general what areas I/we think need to be improved.

I think the provider functionality is very good. One area I think needs work is the player support.
Player support is a bit more difficult to setup. Especially as many folks have multiple different types of players.
In my case I have bluetooth devices, android tablets, window computers, and a couple of linux servers I would like to play music thru. They have not been easy to setup.

Here's my list so far:
1. A Dashboard card for most of the functionality.
2. Better editing/creation of play list. Copy an existing playlist.
3. Better Docs that explain how to use Music Assistant.  Also, more info how to setup players.
4. More configuration of the the existing web page. Like position, colors, etc. Maybe a theme style? 
5. Support for using Music Assistant remotely, not on the home network. Like on a cell phone
6. Android/IOS app

**Comment by OzGav** [comment]
On point 3 what have you found in the docs and how can it be improved.

**Reply by OzGav** [reply]
Any chance of some more info?
